### ISSUE TITLE
alpine linux (docker image) libc issue: __fprintf_chk: symbol not found

### ISSUE THREAD

**Issue by paxapy** [anibali/pywebp#15]
hi. 
i think probably it's not a right place to ask questions like this, but you may know how to help.
i'm trying to use wepb with docker python:3.8.0-alpine image and got this error when trying to import webp:
```
Error relocating /usr/local/lib/python3.8/site-packages/_webp.abi3.so: __fprintf_chk: symbol not found
```
any ideas?

**Comment by anibali** [comment]
I'm not 100% sure, but it could be a related to the fact that Alpine uses musl for its libc implementation. This can cause compatibility issues with native libraries built using a different libc implementation, which includes the prebuilt webp libraries pulled in via Conan for this project (see https://github.com/conan-io/conan/issues/3972). Unfortunately I don't have a good solution for you, though.

**Comment by paxapy** [comment]
ok, but what i'm confused about is that there is actually libwepb implementation for alpine: https://pkgs.alpinelinux.org/package/v3.8/main/x86/libwebp
it's just incompatible with this pywebp package? may be there is a chance i can workaround by using pillow webp converting or it's better just to try?)
anyways thanks for reply, maybe i'll find a way to fix it..

**Comment by anibali** [comment]
This Python package doesn't use the system libwebp, since it gets packaged a bit differently on different distros (eg if I recall correctly, the Ubuntu package is missing stuff). Instead, it uses Conan to grab libwebp.

**Comment by paxapy** [comment]
ah thanks now i finally got what is have to do with conan.io)
okay, so i think for my purposes (converting png to webp) i'll be good with pillow and system libwebp, but in general, i think it's may worth to dig what's going on there because it's seems like using alpine with docker becoming common approach

**Comment by sayhi12345** [comment]
Hi, Is there any updates on this issue yet?

**Comment by anibali** [comment]
Unfortunately not, still waiting on a good solution to https://github.com/conan-io/conan/issues/3972.

**Comment by anibali** [comment]
Should be fixed in v0.2.0.
